src/card.rs

```diff
@@ -0,0 +1,28 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Copy, Clone, Debug)]
+pub enum Suit {
+    Club,
+    Spade,
+    Heart,
+    Diamond,
+}
+
+// TODO: consider refactoring the code with modules. Jack.
```

Comment:
What do you mean by this? Haven't you done this in this pr?

Reply:
Oh yeah you make a good point, I'll remove that

Reply:
Done